src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -359,22 +382,22 @@ void AssimpImporter::doOpenData(const Containers::ArrayView<const char> data) {
         /* Store first possible texture index for this material, next textures
            use successive indices. */
         _f->textureIndices[mat] = textureIndex;
-        for(std::size_t i = 0; i != mat->mNumProperties; ++i) {
+        for(std::size_t j = 0; j != mat->mNumProperties; ++j) {
```

Comment:
Huh. I wonder what kind of bugs this caused :O

Reply:
Probably nothing, since it only shadowed `i` and the original `i` wasn't being used from what I can tell. I forgot to update the comments in the loop, will fix that in another commit.

Reply:
> the original i wasn't being used from what I can tell.

Ah! I thought the inner loop was incrementing the original variable (which would be crazy), missed the redeclaration. :)